src/managers/Queue.ts

```diff
@@ -412,17 +418,19 @@ export class Queue {
     setVolume(volume: number) {
         if(this.destroyed)
             throw new DMPError(DMPErrors.QUEUE_DESTROYED);
+        if(!this.connection)
+            throw new DMPError(DMPErrors.NO_VOICE_CONNECTION);
 
         this.options.volume = volume;
         return this.connection.setVolume(volume);
     }
 
     /**
      * Returns current playing song
-     * @type {Song}
+     * @type {?Song}
      */
-    get nowPlaying() {
```

Comment:
The type annotation is needed here because TS otherwise infers the return type to be `Song` — `this.songs[0]` has the type `Song` not `Song | undefined` because the [`noUncheckedIndexedAccess`](https://www.typescriptlang.org/tsconfig#noUncheckedIndexedAccess) TS option is not enabled.